--- libmcrypt-2.5.7/lib/libmcrypt.m4	2006-10-08 09:04:36.000000000 -0400
+++ libmcrypt-2.5.7/lib/libmcrypt.m4_ORIG	2006-10-08 08:57:00.000000000 -0400
@@ -14,7 +14,7 @@
 dnl AM_PATH_LIBMCRYPT([MINIMUM-VERSION, [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
 dnl Test for libmcrypt, and define LIBMCRYPT_CFLAGS and LIBMCRYPT_LIBS
 dnl
-AC_DEFUN(AM_PATH_LIBMCRYPT,
+AC_DEFUN([AM_PATH_LIBMCRYPT],
 [dnl
 dnl Get the cflags and libraries from the libmcrypt-config script
 dnl
@@ -164,7 +164,7 @@
         printf("*** of LIBMCRYPT, but you can also set the LIBMCRYPT_CONFIG environment to point to the\n");
         printf("*** correct copy of libmcrypt-config. (In this case, you will have to\n");
         printf("*** modify your LD_LIBRARY_PATH enviroment variable, or edit /etc/ld.so.conf\n");
-        printf("*** so that the correct libraries are found at run-time))\n");
+        printf("*** so that the correct libraries are found at run-time)\n");
       }
     }
   return 1;
